.github/libs/GithubUtils.ts

```diff
@@ -13,17 +13,7 @@ import {throttling} from '@octokit/plugin-throttling';
 import {isEmptyObject} from '@src/types/utils/EmptyObject';
 import arrayDifference from '@src/utils/arrayDifference';
 import CONST from './CONST';
-
-const GITHUB_BASE_URL_REGEX = new RegExp('https?://(?:github\\.com|api\\.github\\.com)');
```

Comment:
is this not used here?

Reply:
It's used in `constants.ts` file, I moved it there